src/client.js

```diff
@@ -76,6 +76,10 @@ function Client(manager, name, config) {
 		id: id++,
 		name: name,
 		networks: [],
+		push: { // TODO: This should not ship
```

Comment:
How do other systems handle this? I feel like if someone sends me 5 PMs quickly, we should rate limit them. At least, if Telegram was doing that to me, I'd be pissed lol.
I have noticed though that it only shows up to a certain amount, then just rings but doesn't show (or show "N new messages") but that might be handled by the OS itself.